api/models.go

```diff
@@ -206,7 +214,32 @@ type MobileSearchResponseItem struct {
 	TagUid         *string    `json:"tagId,omitempty"`
 	Title          string     `json:"title"`
 	Image          *string    `json:"image,omitempty"`
-	Views          *int64     `json:"views,omitempty"`
+	ViewsType
```

Comment:
put embedded types at the beginning of the struct fields

Reply:
you mean this way?